src/Models/topological_elements.jl

```diff
@@ -1,15 +1,12 @@
-export Junction
-export PressureZone
-
 
 struct Junction
     number::Int
     name::String
     elevation::Real
-    head::Union{Nothing,Float64}
-    demand::Any 
-    minimum_pressure::Float64
-    coordinates::Union{NamedTuple{(:lat,:lon), Tuple{Float64,Float64}},Nothing}
+    head::Union{Nothing,Float32}
```

Comment:
Why Float32 here? Use Float64 everywhere if there isn't a specific reason to use Float32.

Reply:
Got it! WNTR simulation exports head, headloss, flow, and demand in Float32 and I converted them in the parser to Float64.

Reply:
I'd recommend using Float64 wherever possible.

Reply:
Fixed it. Thank you!